### PR TITLE
Wrong link in doc, also directs to bad URL #26823

### PR DESCRIPTION
**Pull request by helena**
https://github.com/akka/akka/issues/26823 The url changed, this updates it.

### CODE REVIEW

**Review by raboof — APPROVED** [review]

File: akka-docs/src/main/paradox/cluster-routing.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ The easiest way to run **Router Example with Group of Routees** example yourself
 @java[@extref[Akka Cluster Sample with Java](ecs:akka-samples-cluster-java)]
 together with the tutorial. It contains instructions on how to run the **Router Example with Group of Routees** sample.
 The source code of this sample can be found in the
-@scala[@extref[Akka Samples Repository](samples:akka-sample-cluster-scala)]@java[@extref[Akka Samples Repository](samples:akka-sample-cluster-java)].
+@scala[@extref[Akka Samples Repository](samples:akka-samples-cluster-scala)]@java[@extref[Akka Samples Repository](samples:akka-samples-cluster-java)].
```
Comment (patriknw) [inline akka-docs/src/main/paradox/cluster-routing.md]: I think the originally intended link here was to https://github.com/akka/akka-samples/tree/2.5/akka-sample-cluster-scala

It was changed in https://github.com/akka/akka/commit/91cbd6ed99c30b5179f24519de50009c863d5063, but without changing all usages.

Here we link to both a download with `extref ecs` and to the source. I suggest that we change this to only link to `extref samples`, something like this

```
The easiest way to run **Router Example with Group of Routees** example yourself is to try the
@scala[@extref[Akka Cluster Sample with Scala](samples:akka-samples-cluster-scala)]@java[@extref[Akka Cluster Sample with Java](samples:akka-samples-cluster-java)].
It contains instructions on how to run the **Router Example with Group of Routees** sample.
```

Same kind of change at other places.

Comment (helena) [reply akka-docs/src/main/paradox/cluster-routing.md]: @patriknw you want to remove the repo links/text?

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
@helena Let's backport this to release-2.5 branch. Would you like to do it or should I?

**Comment by helena** [comment]
@patriknw If you can do this one, then I'll see what's done and can back port from here on?

**Comment by helena** [comment]
I see the back port doc in https://github.com/akka/akka/blob/master/CONTRIBUTING.md

**Comment by patriknw** [comment]
Thanks. This is the way I do it:
* label this PR with `to-be-backported`
* mark this PR with Milestone `2.6.0-M1`
* mark the issue with Milestone `2.5.22`
* cherry pick to release-2.5
  * git checkout release-2.5
  * git pull
  * git cherry-pick `<commit>`
  * open PR, target release-2.5
* label that PR with `backport`
* merge backport PR after validation (no need for full PR reviews)
* close issue

Feel free to add to CONTRIBUTING.md if you find it useful

**Comment by helena** [comment]
Very, will add it, thank you.

**Pull request merged**